src/main/java/com/monumental/triggers/MonumentTrigger.java

```diff
@@ -17,7 +17,12 @@ void beforeInsert(Monument record) {
         if (record.getAddress() != null) {
             return;
         }
-        String address = googleMapsService.getAddressFromCoordinates(record.getLat(), record.getLon());
+
+        if (record.getPoint() == null) {
+            return;
+        }
+
+        String address = googleMapsService.getAddressFromCoordinates(record.getPoint().getY(), record.getPoint().getX());
```

Comment:
Might make sense to add `getLat` and `getLon` to `Monument` as a shortcut for these `getY` and `getX` functions